sw/device/silicon_creator/manuf/base/BUILD

```diff
@@ -366,15 +371,19 @@ _FT_PROVISIONING_HARNESS = "//sw/host/provisioning/ft"
         },
         fpga = fpga_params(
             timeout = "long",
+            assemble = "{ft_personalize}@{slot_a} {rom_ext}@{slot_b}",
```

Comment:
while we are at it, why not include the RW as well? Especially since SPI rescue is not yet functional, how do we get to use the chip after personalization if the only valid image on it is ROM_EXT_B?

Reply:
I would prefer we made this a SKU configuration, i.e., add a field `transport_image` to [this dictionary](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/manuf/base/provisioning_inputs.bzl;drc=118a7713bbc060e58832437f071657abfe203f6f;l=28) that is the firmware image that will get placed in slot B. that way the Sival SKU can only place the ROM_EXT, and various other product SKUs can place a combo of ROM_EXT + their Owner firmware image. Does that make sense?

Reply:
I think we can do this cleanup in a follow up PR.

Reply:
#24971 will fix this.

Reply:
> I would prefer we made this a SKU configuration, 

sure, as long as there is a way to do this for the Cros case.